osu.Game/Overlays/Mods/ModCustomisationPanel.cs

```diff
@@ -161,13 +164,13 @@ private void updateDisplay()
                 content.AutoSizeDuration = 400;
                 content.AutoSizeEasing = Easing.OutQuint;
                 content.AutoSizeAxes = Axes.Y;
-                content.FadeEdgeEffectTo(0.25f, 120, Easing.OutQuint);
+                content.FadeIn(120, Easing.OutQuint);
             }
             else
             {
                 content.AutoSizeAxes = Axes.None;
                 content.ResizeHeightTo(header_height, 400, Easing.OutQuint);
-                content.FadeEdgeEffectTo(0f, 400, Easing.OutQuint);
+                content.FadeOut(400, Easing.OutSine);
```

Comment:
Yup, I personally picked `OutSine` over `OutQuint` because the latter makes the content vanish quite abruptly when the mod customisation panel is too long.